content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

```diff
@@ -66,13 +66,45 @@ MTM-47277</td>
 <td>MTM-47879</td>
 </tr>
 
+<tr>
+<td>Administration</td>
+<td>Files of shared applications are now accessible to all subtenants of the owner tenant. This supports a "self-service" approach for web applications (subtenants can decide if they want to install a shared application) as opposed to the currently used concept (only a parent tenant can decide if a web application gets subscribed to a subtenant).

```

Comment:
```suggestion
<td>Files of shared applications are now accessible to all subtenants of the owner tenant. This supports a "self-service" approach for web applications (subtenants can decide if they want to install a shared application) as opposed to the current approach (only a parent tenant can decide if a web application gets subscribed to a subtenant).
```

the information in parentheses could be added as "where" relative clauses instead.